neqo-crypto/build.rs

```diff
@@ -357,8 +357,7 @@ fn setup_standalone(nss: &str) -> Vec<String> {
     // $NSS_DIR/../dist/
     let nssdist = nss.parent().unwrap().join("dist");
     println!("cargo:rerun-if-env-changed=NSS_TARGET");
-    let nsstarget = env::var("NSS_TARGET")
-        .unwrap_or_else(|_| fs::read_to_string(nssdist.join("latest")).unwrap());
+    let nsstarget = env::var("NSS_TARGET").unwrap_or_else(|_| String::from("Debug"));
```

Comment:
I don't think we should default to `Debug` here.

```suggestion
    let nsstarget = env::var("NSS_TARGET").unwrap_or_else(|_| String::from("Release"));
```